scenario/BulkerScenario.ts

```diff
@@ -162,7 +163,7 @@ scenario(
 scenario(
   'Comet#bulker > (non-WETH base) all actions in one txn',
   {
-    filter: async (ctx) => await isBulkerSupported(ctx) && await isRewardSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, { network: 'linea-goerli' }]),
+    filter: async (ctx) => await isBulkerSupported(ctx) && await isRewardSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, { network: 'linea-goerli' }, {deployment: 'weth-lrt', network: 'mainnet'}]),
```

Comment:
We suppose, that if WETH deployment was skipped before, we should skip it as well.